mobile/src/board/entity.tsx

```diff
@@ -0,0 +1,60 @@
+export enum GameState {
```

Comment:
Don't use TypeScript's `enum` type. 

https://www.executeprogram.com/blog/typescript-features-to-avoid